### DISCUSSION TITLE
Is it possible to take out textures out of a spritesheet for a 3D model?

### DISCUSSION THREAD

**Discussion by leha-code** [pokepetter/ursina#435] [Q&A]
Is it possible to have a spritesheet with many different textures to have specific textures applied to a 3D model?

**Comment by RedHenDev** [comment]
Yes.

You can access the uvs of your model and scale the model's texture according to your spritesheet/texture atlas.

This video shows you how in terms of Minecraft-style terrain with block textures loaded from one texture atlas:  https://youtu.be/fofR4yzTCdA

**Reply by leha-code** [reply]
This isn't how I wanted to load the images. I want to be able to load the textures from an image [like this](https://0fps.files.wordpress.com/2013/07/terrain.png). Is there any way to do that?
